Mathlib/Combinatorics/Additive/QuotientInterGrowth.lean

```diff
@@ -0,0 +1,80 @@
+/-
+Copyright (c) 2024 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.Group.Pointwise.Finset.Basic
+import Mathlib.Combinatorics.Enumerative.DoubleCounting
+import Mathlib.GroupTheory.QuotientGroup.Defs
+
+/-!
+# Growth in the quotient and intersection with a subgroup
+
+For a group `G` and a subgroup `H ≤ G`, this file upper and lower bounds the growth of a finset by
```

Comment:
It looks like you only lower bound the growth currently?

Reply:
`card_pow_quotient_mul_pow_inter_subgroup_le` is the lower bound, and `le_card_quotient_mul_sq_inter_subgroup` is the upper bound (just set `A := A ^ n`)